torch/_torch_docs.py

```diff
@@ -5942,6 +5942,32 @@ def merge_dicts(*dicts):
     tensor([ 0,  2,  0,  1])
 """.format(**single_dim_common))
 
+add_docstr(torch.argwhere,
+           r"""
+argwhere(input) -> LongTensor
+
+Return the indices of array elements that are non-zero, grouped by element.
```

Comment:
I found the "grouped by element" language from numpy's documentation to be pretty informal and hard to understand. Looks like I'm also not the first one: https://stackoverflow.com/questions/52400354/how-to-understand-the-np-argwhere-function/52400485.

It seems that "grouped by element" is stated in numpy.argwhere's documentation in reference to numpy.nonzero's behavior (https://numpy.org/doc/stable/reference/generated/numpy.nonzero.html), which groups by dimension instead.

For our purposes it seems clearer to reuse/refer to the language from torch.nonzero's documentation of its as_tuple=False case: https://pytorch.org/docs/stable/generated/torch.nonzero.html.

Reply:
That makes sense. Thanks!